base/internal/message.hpp

```diff
@@ -28,6 +28,9 @@ std::string DebugPrint(std::string const & t);
 inline std::string DebugPrint(char const * t);
 inline std::string DebugPrint(char * t) { return DebugPrint(static_cast<char const *>(t)); }
 inline std::string DebugPrint(char t);
+inline std::string DebugPrint(char32_t const * t);
+inline std::string DebugPrint(char32_t * t) { return DebugPrint(static_cast<char32_t const *>(t)); }
+inline std::string DebugPrint(char32_t t);
```

Comment:
I don't have c++20 yet but suspect that we can remove it for now.

Reply:
Please see my response below.